src/postprocessors/NekPostprocessor.C

```diff
@@ -24,11 +24,18 @@ InputParameters
 NekPostprocessor::validParams()
 {
   InputParameters params = GeneralPostprocessor::validParams();
+
+  MooseEnum mesh("fluid solid all", "fluid");
```

Comment:
I think this default should also be `all` in order to match the previous behavior.